src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -181,6 +191,17 @@ public uint GetMaxValidUntilBlockIncrement(IReadOnlyStore snapshot)
             return (uint)(BigInteger)snapshot[_maxValidUntilBlockIncrement];
         }
 
+        /// <summary>
+        /// Gets the length of the chain accessible to smart contracts.
+        /// </summary>
+        /// <param name="snapshot">The snapshot used to read data.</param>
+        /// <returns>MaxTraceableBlocks value.</returns>
+        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates)]
```

Comment:
```suggestion
        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 14, RequiredCallFlags = CallFlags.ReadStates)]
```

Reply:
Check out the https://github.com/neo-project/neo/pull/3898#discussion_r2048515368.